testing/scenario_app/android/run_gradle.py

```diff
@@ -11,13 +11,18 @@
 import sys
 import subprocess
 
+SCRIPT_PATH = os.path.dirname(os.path.realpath(__file__))
+ANDROID_HOME = os.path.join(SCRIPT_PATH, '..', '..', '..', '..', 'third_party',
+    'android_tools', 'sdk')
+
 def main():
   BAT = '.bat' if sys.platform.startswith(('cygwin', 'win')) else ''
   android_dir = os.path.abspath(os.path.dirname(__file__))
   gradle_bin = os.path.join('.', 'gradlew%s' % BAT)
   result = subprocess.check_output(
     args=[gradle_bin] + sys.argv[1:],
     cwd=android_dir,
+    env=dict(os.environ, ANDROID_HOME=ANDROID_HOME),
```

Comment:
Can we trip an assertion here when our version of ANDROID_HOME cannot be resolved? That way, if we move this script around, we won't end up with a silently depending on the hosts environment variables.

Reply:
Python noob: Will this override the ANDROID_HOME in the existing dictionary?

Reply:
Done

Reply:
(although I think it'd also trip anyway when gradlew would beunable to find it, but failing faster is better)

Reply:
I was more concerned about using a potentially unexpected version of Android at ANDROID_HOME.

Reply:
Yes - that will add or overwrite ANDROID_HOME in os.environ

Reply:
The Android plugin ensures that it can locate the version specified in `compileSdkVersion`/`targetSdkVersion` since it needs to find the right jar with the APIs the app may use.